src/style.css

```diff
@@ -0,0 +1,49 @@
+body {
+  margin: 0;
+}
+
+h1 {
+  margin: 0;
+}
+
+.toptop-heading {
+  display: flex;
+  justify-content: space-between;
+}
+
+.container {
+  padding: 20px;
+  background-color: white;
+  border: 2px solid gray;
+  margin: 20px;
+  display: flex;
+  justify-content: space-between;
+}
+
+ul {
+  margin: 0;
+}
+
+ul li {
+  display: flex;
+  border: 2px solid #f7f7f7;
+  list-style-type: none;
+  justify-content: space-between;
+}
+
+button {
+  color: azure;
+  display: block;
+  background-color: aliceblue;
+  padding: 12px;
+  outline: none;
+}
+
+.fa-solid {
+  color: #e5e9f0;
+}
+
+#form {
+  display: flex;
+  flex-wrap: nowrap;
+}
```

Comment:
| Your TodoList | Wireframe |
| --- | --- |
| ![image](https://github.com/techmoves/todo-list/assets/93039359/7ab4549c-3402-4358-aad7-2b116bdc5380) | ![image](https://github.com/techmoves/todo-list/assets/93039359/93a297d6-81f6-440b-94ae-c2e27f1df110) |

- [ ] It would be nice if you follow the design to make your app look attractive and professional 😊 because now your app is not following the given design properly 😨 Kindly make your design look professional 😊